en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -461,3 +461,21 @@ Note the NED velocity innovations and NE position innovations 'flat-line' after
 Note that after 10 seconds without GPS data, the EKF reverts back to a static position mode using the last known position and the NE position innovations start to change again.
 
 ![GPS Data Loss - in SITL](../../assets/ecl/gps_data_loss_-_velocity_innovations.png)
+
+### Barometer Ground Effect Compensation
+
+If the vehicle has the tendency during landing to climb back into the air when close to the ground, the most likely cause is barometer ground effect.
+
+This is caused when air pushed down by the propellers hits the ground and creates a high pressure zone below the drone.
+The result is a lower reading of pressure altitude, leading to an unwanted climb being commanded.
+The figure below shows a typical situation where the ground effect is present. 
+Note how the barometer signal dips at the beginning and end of the flight.
+
+![Barometer ground effect](../../assets/ecl/gnd_effect.png)
+
+You can enable *ground effect compensation* to fix this problem:
+- From the plot (like the one above) estimate the magnitude of the barometer dip during takeoff or landing.
+- Then set the parameter [EKF2_GND_EFF_DZ](../advanced_config/parameter_reference.md#EKF2_GND_EFF_DZ) to that value and add a 20 percent margin.
```

Comment:
Might be worth showing this for the graph - looking at this I would have said the dip was about 7, so set EKF2_GND_MAX_HGT to 8.5 ish - is that reasonable?

Reply:
@RomanBapst I updated this to my taste. Please check you're happy with this. Also see ^^^

Reply:
@RomanBapst ^^^ Please do final check so this can be merged.